snippets/sdn-deprecation-statement-managed.adoc

```diff
@@ -0,0 +1,34 @@
+// Text snippet included in the following assemblies:
+//
+// * networking/about-networking.adoc
+// * networking/openshift_sdn/assigning-egress-ips.adoc
+// * networking/openshift_sdn/editing-egress-firewall.adoc
+// * networking/openshift_sdn/enabling-multicast.adoc
+// * networking/openshift_sdn/migrate-to-openshift-sdn.adoc
+// * networking/openshift_sdn/multitenant-isolation.adoc
+// * networking/openshift_sdn/removing-egress-firewall.adoc
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+// * networking/ovn_kubernetes_network_provider/viewing-egress-firewall-ovn.adoc
+// * networking/openshift_sdn/about-openshift-sdn.adoc
+// * networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc
+// * networking/openshift_sdn/configuring-egress-firewall.adoc
+//
+// Text snippet included in the following modules:
+//
+// * modules/nw-networking-glossary-terms.adoc
+// * modules/nw-ovn-kubernetes-migration-about.adoc
+// * modules/optimizing-mtu-networking.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
```

Comment:
This box has some factual and logical errors. I've fixed them:

Red Hat OpenShift Service on AWS follows the life cycle of SDN according to OpenShift Core Platform. Accordingly, 

- SDN is deprecated for clusters as of OpenShift version 4.14.

-  Clusters that already use the OpenShift SDN plugin can use the plugin after they are upgraded to OpenShift version 4.11 and above.
     Clusters can be upgraded to OpenShift versions up to version 4.16.

- Clusters running on OpenShift 4.16 
    Clusters using OpenShift version 4.16 cannot upgrade if the clusters are using the SDN plugin.
    The SDN plugin will be discontinued in OpenShift version 4.17.